### PR TITLE
Context routes

### PR DESCRIPTION
**Pull request by wwajerowicz**

### CODE REVIEW

File: modules/service/src/main/scala/ch/epfl/bluebrain/nexus/kg/service/persistence/TaggingAdapter.scala [hunk 1]
```diff
@@ -12,11 +13,12 @@ import ch.epfl.bluebrain.nexus.kg.core.schemas.SchemaEvent
 class TaggingAdapter extends WriteEventAdapter {
 
   override def manifest(event: Any): String = ""
-
+//TODO Contexts
```
Comment (umbreak) [inline modules/service/src/main/scala/ch/epfl/bluebrain/nexus/kg/service/persistence/TaggingAdapter.scala]: remove comment

**Comment by wwajerowicz** [comment]
retest this please

**Comment by wwajerowicz** [comment]
retest this please

**Comment by wwajerowicz** [comment]
retest this please

**Comment by umbreak** [comment]
retest this please

**Comment by umbreak** [comment]
retest this please

**Pull request merged**